### PR TITLE
added support for datastream

### PR DESCRIPTION
**Pull request by ferronrsmith**

### CODE REVIEW

File: bin/multielasticdump [hunk 1]
```diff
@@ -339,15 +340,32 @@ if (options.direction === 'dump') {
...
+          indexes = Object.keys(response)
+        }
+        matchedIndexes = indexes.filter(index => {
+          const aliases = Object.keys(response[index].aliases || {})
+          return matchRegExp.test(index) || aliases.some(alias => matchRegExp.test(alias))
+        })
+        break
+      }
+      case 'datastream': {
+        matchedIndexes = response.data_streams.filter(stream => matchRegExp.test(stream)).map(stream => stream.name)
```
Comment (vfronterotta) [inline bin/multielasticdump:361]: Change matchRegExp.test(stream)) to matchRegExp.test(stream.name))

Comment (ferronrsmith) [reply bin/multielasticdump]: this is fixed.

File: README.md [hunk 2]
```diff
@@ -778,6 +779,8 @@ i.e analyzer,alias types are ignored by default
 
 `--ignoreChildError` allows multi-elasticdump to continue if a child throws an error.
 
+`--matchType`  allows multi-elasticdump to fetch index name from elasticsearch alias endpoint of data_streams
```
Comment (vfronterotta) [inline README.md:782]: Can we change to the following?
allows multi-elasticdump to fetch indices from the specified elasticsearch endpoint. Two options are supported - \`alias, datastream\`

Comment (ferronrsmith) [reply README.md]: fixed

**Pull request merged**